gui/src/common/regex-helpers.ts

```diff
@@ -41,7 +41,7 @@ export const passwordRegex = helpers.regex(
 );
 export const passwordTooltipMessage =
   // tslint:disable-next-line
-  "Password must contain: Minimum 8 characters, One uppercase letter, One lowercase letter, One special character, One number";
+  "Password must contain: Minimum 8 characters, One uppercase letter, One lowercase letter, One special character, and One number";
```

Comment:
Please remove comma before and

Reply:
that was the requirement, document team suggested to add comma